src/test/java/org/hyperledger/fabric/sdk/NetworkConfigTest.java

```diff
@@ -318,6 +320,61 @@ public void testLoadFromConfigFileYamlNOOverrides() throws Exception {
 
     }
 
+    @Test
+    public void testLoadFromConfigFileJsonNOOverridesClientTls() throws Exception {
+
+        File f = NETWORK_CONFIG_CLIENT_TLS_JSON.toFile();
+        NetworkConfig config = NetworkConfig.fromJsonFile(f);
+
+        assertNotNull(config);
+
+        String peer0 = "peer0.org1.example.com";
+        String peer1 = "peer1.org1.example.com";
+        // Check JsonPeers' properties
+        Properties peer0Properties = config.getPeerProperties(peer0);
+        assertEquals(peer0Properties.getProperty("clientCertFile"), "./tls/sign.pem");
+        assertEquals(peer0Properties.getProperty("clientKeyFile"), "./tls/key.pem");
```

Comment:
I think it would be better to use constants from the code for these values.

Reply:
Replaced after moving constants to NetworkConfig.